staging/src/k8s.io/apimachinery/pkg/util/proxy/upgradeaware.go

```diff
@@ -282,6 +289,11 @@ func (h *UpgradeAwareHandler) tryUpgrade(w http.ResponseWriter, req *http.Reques
 		backendConn, rawResponse, err = utilnet.ConnectWithRedirects(req.Method, &location, clone.Header, req.Body, utilnet.DialerFunc(h.DialForUpgrade), h.RequireSameHostRedirects)
 	} else {
 		klog.V(6).Infof("Connecting to backend proxy (direct dial) %s\n  Headers: %v", &location, clone.Header)
+		if h.OverrideRequestHost {
+			// exchanging req.Host with the backend location is necessary for backends that act on the HTTP host header (e.g. API gateways),
```

Comment:
I believe no need to duplicate the comment from line 232. git blame also would help identify which issue is about.

Reply:
@dougsland done!